src/OrbitGl/SamplingReportDataView.cpp

```diff
@@ -278,27 +278,43 @@ void SamplingReportDataView::OnContextMenu(const std::string& action, int menu_i
   }
 }
 
-void SamplingReportDataView::UpdateSelectedAddressesAndTid(const std::vector<int>& indices) {
+void SamplingReportDataView::UpdateSelectedIndicesAndFunctionIds(
+    const std::vector<int>& selected_indices) {
+  selected_indices_.clear();
+  selected_function_ids_.clear();
+  for (int row : selected_indices) {
+    selected_indices_.insert(indices_[row]);
+    selected_function_ids_.insert(GetSampledFunction(row).absolute_address);
+  }
+}
+
+void SamplingReportDataView::UpdateSelectedIndicesOnFunctionsChanged() {
```

Comment:
Nit: All the new method start with "Update" which is not wrong. But I would prefer for this function a name that makes it clear that this function is different. Maybe something like: `RestoreSelectedIndicesAfterFunctionsChanged`

Reply:
Thanks for suggesting the name! Changed to use `RestoreSelectedIndicesAfterFunctionsChanged`.